aspnetcore/fundamentals/dependency-injection.md

```diff
@@ -84,15 +84,20 @@ This interface is implemented by a concrete type, `MyDependency`:
 The container resolves `ILogger<TCategoryName>` by taking advantage of [(generic) open types](/dotnet/csharp/language-reference/language-specification/types#open-and-closed-types), eliminating the need to register every [(generic) constructed type](/dotnet/csharp/language-reference/language-specification/types#constructed-types):
 
 ```csharp
+// This code provided by the framework.
 services.AddSingleton(typeof(ILogger<>), typeof(Logger<>));
```

Comment:
Maybe we should remove this as an example since it's never relevant in practice

Reply:
Will do.
BTW, this is not ready for review but comments like that are welcome.